includes/AppLeave.php

```diff
@@ -158,8 +161,8 @@
 	//echo "Sending application to $appr";
 	
 	$destDB = str_replace('.','$',$appr);
-	$stmt = $sqlConn->prepare("INSERT INTO $destDB VALUES (?,?)");
-	$stmt->bind_param('ss',$id,$appTy);
+	$stmt = $sqlConn->prepare("INSERT INTO $destDB VALUES (?,?,?)");
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->